cli/actions/resource.go

```diff
@@ -0,0 +1,47 @@
+package actions
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"go.uber.org/zap"
+)
+
+type ApplyArgs struct {
+	File string
+}
+
+type ResourceActions interface {
```

Comment:
Resource interface which is going to establish the methods for each supported resource